integration/simulation/simulation_geth_in_mem_erc20_test.go

```diff
@@ -0,0 +1,65 @@
+package simulation
```

Comment:
is the rename necessary? It makes it hard to compare

Reply:
Changed it back 👍